### DISCUSSION TITLE
Scaling factor explanation request

### DISCUSSION THREAD

**Discussion by DharaniSureshBabu** [cu-esiil-edu/2024-data-short-course#94] [Q&A]
@eculler could you please explain how we came up with these values ? 

doy_start = -19
doy_end = -12

**Comment by eculler** [comment]
@DharaniSureshBabu yes, those values are counting characters from the end of the string (because they're negative). We're using them to pull the date out of the file name

**Reply by DharaniSureshBabu** [reply]
Thank you!
